src/libraries/Microsoft.PowerFx.Connectors/Execution/FormulaValueSerializer.cs

```diff
@@ -72,6 +73,29 @@ private void WriteObject(string objectName, OpenApiSchema schema, IEnumerable<Na
                 WriteProperty(property.Key, property.Value, namedValue.Value);
             }
 
+            if (!schema.Properties.Any() && fields.Any())
+            {
+                foreach (NamedValue nv in fields)
+                {
+                    WriteProperty(
+                        nv.Name,
+                        new OpenApiSchema()
+                        {
+                            Type = nv.Value.Type._type.Kind switch
+                            {
+                                DKind.Number => "number",
```

Comment:
>"number" [](http://example.com/codeflow?start=48&length=8)

nit - (future PR is fine) - we repeat these string literals alot. 
Can we encapsulate somewhere.
public const string NumberType = "number";

Reply:
Can do that in another PR but we would need a prefix
Ex: OpenApiNumberType (maybe there is an existing one..)

Reply:
Just put them in a static class - that's your prefx.